### PR TITLE
Add issues-collection examples

### PR DESCRIPTION
**Pull request by RobiNino**

### CODE REVIEW

File: jenkins-examples/pipeline-examples/declarative-examples/README.md [hunk 1]
```diff
@@ -5,13 +5,14 @@ To set up Jenkins to use the example, read [this](https://github.com/jfrog/proje
...
 * The [exclude-patterns-download-example](exclude-patterns-download-example) demonstrates how to exclude certain files while downloading.
 * The [exclude-patterns-upload-example](exclude-patterns-upload-example) demonstrates how to exclude certain files while uploading.
 * The [gradle-example-ci-server](gradle-example-ci-server) resolves dependencies, deploys artifacts and publishes build-info to Artifactory for a Gradle build. Unlike the [gradle-example](gradle-example), this examples assumes that the Gradle Artifactory Plugin in not applied to the Gradle build script.
 * The [gradle-example](gradle-example) resolves dependencies, deploys artifacts and publishes build-info to Artifactory for a Gradle build. Unlike the [gradle-example-ci-server](gradle-example-ci-server), this examples assumes that the Gradle Artifactory Plugin in already applied in the Gradle build script.
 * The [interactive-promotion-example](interactive-promotion-example) demonstrates how to promote a build in Artifactory after the build finished.
+* The [issues-collection-example](issues-collection-example) demonstrates how to collect the list of tracked project issues and them to the build-info.
```
Comment (yahavi) [inline jenkins-examples/pipeline-examples/declarative-examples/README.md]:
```suggestion
* The [issues-collection-example](issues-collection-example) demonstrates how to collect the list of tracked project issues and add them to the build-info.
```

File: jenkins-examples/pipeline-examples/scripted-examples/README.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ To set up Jenkins to use the example, read [this](https://github.com/jfrog/proje
 * The [gradle-deploy-example](gradle-deploy-example) demonstrates how to defer the build artifacts deployment to a separate stage.
 * The [gradle-example-ci-server](gradle-example-ci-server) resolves dependencies, deploys artifacts and publishes build-info to Artifactory for a Gradle build. Unlike the [gradle-example](gradle-example), this examples assumes that the Gradle Artifactory Plugin in not applied to the Gradle build script.
 * The [gradle-example](gradle-example) resolves dependencies, deploys artifacts and publishes build-info to Artifactory for a Gradle build. Unlike the [gradle-example-ci-server](gradle-example-ci-server), this examples assumes that the Gradle Artifactory Plugin in already applied in the Gradle build script.
+* The [issues-collection-example](issues-collection-example) demonstrates how to collect the list of tracked project issues and them to the build-info.
```
Comment (yahavi) [inline jenkins-examples/pipeline-examples/scripted-examples/README.md]:
```suggestion
* The [issues-collection-example](issues-collection-example) demonstrates how to collect the list of tracked project issues and add them to the build-info.
```

File: jenkins-examples/job-dsl-examples/pipeline-example/JobDSL.groovy [hunk 3]
```diff
@@ -7,7 +7,7 @@ pipelineJob('job-dsl-artifactory-pipeline-example') {
 
     definition {
         cps {
-            script(readFileFromWorkspace('jenkins-examples/pipeline-examples/declarative-example/Jenkinsfile'))
+            script(readFileFromWorkspace('jenkins-examples/pipeline-examples/scripted-examples/declarative-example/Jenkinsfile'))
```
Comment (eyalbe4) [inline jenkins-examples/job-dsl-examples/pipeline-example/JobDSL.groovy]: Was this path changed intentionally? The path looks wrong (...scripted-examples/declarative-example...)

File: jenkins-examples/pipeline-examples/declarative-examples/issues-collection-example/Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                            "keyGroupIndex": "1",
+                            "summaryGroupIndex": "2",
+                            "trackerUrl": "http://my-jira.com/issues",
+                            "aggregate": "true",
+                            "aggregationStatus": "RELEASED"
+                        }
+                    }""",
+                    
+                    // You may alternatively provide a path to a config file, instead of the config itself, by setting:
+                    //configPath: '/path/to/config'
```
Comment (eyalbe4) [inline jenkins-examples/pipeline-examples/declarative-examples/issues-collection-example/Jenkinsfile]: //configPath: '/path/to/config' --> // configPath: '/path/to/config'
(Missing white-space).

File: jenkins-examples/job-dsl-examples/pipeline-example/JobDSL.groovy (hunk 3 above)
Comment (RobiNino) [reply jenkins-examples/job-dsl-examples/pipeline-example/JobDSL.groovy]: Yes, it is an example of a scripted pipeline implemented as a step in a declarative pipeline.
The other path does not exist

**Pull request merged**